tomcat/src/main/java/org/apache/catalina/session/SessionManager.java

```diff
@@ -1,11 +1,16 @@
 package org.apache.catalina.session;
 
-import java.util.HashMap;
-import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
 
 public class SessionManager implements Manager {
 
-    private static final Map<String, Session> SESSIONS = new HashMap<>();
+    private static final SessionManager INSTANCE = new SessionManager();
+    private static final ConcurrentMap<String, Session> SESSIONS = new ConcurrentHashMap<>();
```

Comment:
```suggestion
    private static final Map<String, Session> SESSIONS = new ConcurrentHashMap<>();
```
![image](https://user-images.githubusercontent.com/57438644/189330027-abdcc3f0-5142-42ee-b687-38b8b550f02a.png)
Map으로 하는게 좋아보입니다!

Reply:
SESSIONS의 동시성이 보장되어야 한다는 것은 굉장히 중요하다고 생각했기 때문에 일부러 `ConcurrentMap`으로 선언해주었습니다 😸

Reply:
네네! ConcurrentHashMap 선언해줄 때 인터페이스를 사용해 참조하도록 말씀드린 거였어요!
(Effective Java) 규칙64. 객체는 인터페이스를 사용해 참조하라 부분을 참고해주시면 좋을 것 같습니다 :)